pkg/sql/testdata/postgres_test_expected_diffs_on_pg_catalog.json

```diff
@@ -19,20 +19,6 @@
     "pg_attribute": {
       "attmissingval": null
     },
-    "pg_class": {
```

Comment:
why did these get removed now? shouldn't the test automatically run on each commit to make sure the diff file is correct?

Reply:
seems like we added all missing columns at pg_class and the diff file was outdated. When we re-write the diffs now we can see that there are no diffs on pg_class.